src/string.ts

```diff
@@ -12,4 +12,8 @@ export const capitalize = <S extends string>(s: S): Capitalize<S> =>
 export const uncapitalize = <S extends string>(s: S): Uncapitalize<S> =>
   (S.isEmpty(s) ? s : s[0].toLowerCase() + s.slice(1)) as Uncapitalize<S>
 
+export const prefix = (prefix: string) => (s: string) => `${prefix}${s}`
+
+export const suffix = (suffix: string) => (s: string) => `${s}${suffix}`
```

Comment:
There are too many ways to concatenate strings, and template-string is a good one, but what would you say if we considered string semigroups of `fp-ts` to get the same result without worrying about how?